suite-native/notifications/src/components/TransactionNotificationDescription.tsx

```diff
@@ -17,24 +15,16 @@ const addressContainerStyle = prepareNativeStyle(_ => ({
 }));
 
 export const TransactionNotificationDescription = ({
-    amount,
+    formattedAmount,
     prefix,
     targetAddress,
-    networkSymbol,
 }: TransactionNotificationDescriptionProps) => {
     const { applyStyle } = useNativeStyles();
 
     return (
         <Box flexDirection="row">
-            <CryptoAmountFormatter
-                value={amount}
-                network={networkSymbol}
-                isBalance={false}
-                color="textSubdued"
-                variant="label"
-            />
```

Comment:
Why this change? If you want to dispatch notification for new transaction, you won't have formatted amount there and should be handled here.

Reply:
reverted in 53e695678